### PR TITLE
Add link to PR Dashboard

### PR DESCRIPTION
**Pull request by dims**
Fixes https://github.com/kubernetes/community/issues/550

### CODE REVIEW

File: k8s.io/configmap-nginx.yaml [hunk 1]
```diff
@@ -204,6 +204,7 @@ data:
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-gubernator/triage/index.html redirect;
           rewrite ^/test-health$     http://velodrome.k8s.io/dashboard/db/bigquery-metrics redirect;
+          rewrite ^/pr-dashboard$    https://k8s-gubernator.appspot.com/pr
```
Comment (ixdy) [inline k8s.io/configmap-nginx.yaml]: this needs ` redirect;` at the end of the line

Comment (dims) [inline k8s.io/configmap-nginx.yaml]: oops fixed.

**Comment by ixdy** [comment]
/lgtm

canary cluster passes tests.

**Comment by ixdy** [comment]
deployed: https://go.k8s.io/pr-dashboard

**Pull request merged**